### ISSUE TITLE
False Positive global-require when Subtraction Operation

### ISSUE THREAD

**Issue by xcatliu** [eslint/eslint#8942]
**Tell us about your environment**

* **ESLint Version:** v4.2.0
* **Node Version:** v8.1.2
* **npm Version:** v5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**

Babel-ESLint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
  "rules": {
    "global-require": "error"
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
var a = require('./b') - 1;
```

**What did you expect to happen?**

No warnings

**What actually happened? Please include the actual, raw output from ESLint.**

[eslint] Unexpected require(). (global-require)

**Comment by not-an-aardvark** [comment]
Thanks for reporting. This is very similar to https://github.com/eslint/eslint/issues/8784, so I'm going to close this issue and leave a comment on https://github.com/eslint/eslint/issues/8784 describing this example.
